src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/index.js

```diff
@@ -1,4 +1,6 @@
 // @flow
 import Radio from './Radio';
+import RadioGroup from './RadioGroup';
 
 export default Radio;
```

Comment:
Not sure if we should still add a default export if the element is already in a named export... Haven't done it in the table component either... I think the important thing here would be consistency.

Reply:
remove default